### PR TITLE
CENSORED

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Adds CENSORED.

## Why It's Good For The Game

- Is it..?

### CODE REVIEW

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
No Records.
Records are an essential part of keeping players informed on abnormalities. 
It's not as important when the abnormality doesn't instantly kill you, but on HEs and above it absolutely is essential.
Fix this problem and then I'll approve it.

**Comment by EgorDinamit** [comment]
> Fix this problem and then I'll approve it.

You are not a maintainer, but there is a reason I didn't merge this PR yet anyway.

**Comment by Kitsunemitsu** [comment]
You said every comment would be looked at, and while I'm not a maintainer I couldn't give more of a damn. 
Players rely on records for 100% accurate information on the game. 

Also, I think that me telling you this, the person who created, makes and maintains records for the sake of all players, is a slap in the face.

It's almost like when someone makes and maintains critical systems for player QOL telling them "They're not a maintainer so their approval doesnt mean anything" pisses them off.

**Comment by EgorDinamit** [comment]
> You said every comment would be looked at, and while I'm not a maintainer I couldn't give more of a damn. Players rely on records for 100% accurate information on the game.
> 
> Also, I think that you telling me this, the person who created, makes and maintains records for the sake of all players, is a slap in the face.
> 
> It's almost like when someone makes and maintains critical systems for player QOL telling them "They're not a maintainer so their approval doesnt mean anything" pisses them off.

Okay then, sorry for saying it like that.

**Comment by dsmith328** [comment]
I can say the thing. Fix Merge Conflicts.

File: code/modules/mob/living/simple_animal/abnormality/teth/blood_bath.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	work_damage_type = WHITE_DAMAGE
 	max_boxes = 14
 	ego_list = list(
-		/datum/ego_datum/weapon/bloodbath,
+	//	/datum/ego_datum/weapon/wrist,
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/teth/blood_bath.dm:19]: Feel like this should've been atomized?

File: code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(80, 70, 60, 50, 40),
+						ABNORMALITY_WORK_INSIGHT = list(90, 80, 70, 60, 50),
+						ABNORMALITY_WORK_ATTACHMENT = list(70, 60, 50, 40, 30),
+						ABNORMALITY_WORK_REPRESSION = 0,
+						"Sacrifice" = 999,
+						)
+	work_damage_amount = 14
+	work_damage_type = BLACK_DAMAGE
+
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm:38]: Genuinely it needs EGO, at least the weapon, to be fun to deal with at all. No weapon means there's literally no reward for this havoc-wreaking bastard to be picked. You know of the Fear changes that are in the works as well so I don't need to go into detail about that.

File: code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+	faction = list("censored")
+	speak_emote = list("screeches")
+	attack_verb_continuous = "attacks"
+	attack_verb_simple = "attack"
+	attack_sound = 'sound/abnormalities/censored/mini_attack.ogg'
+	/* Stats */
+	health = 600
+	maxHealth = 600
+	obj_damage = 300
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 1.2, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 1)
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm:148]: At some point I feel like we need to give the manager a "Secret Information" file that holds information for Ordeals and Abnormality "Children" (Bees, CENSOREDs, Perfect Foods, etc)

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/teth/blood_bath.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/abnormality/teth/blood_bath.dm:19]: It's a fix, weapon is for some reason just not here.

**Comment by EgorDinamit** [comment]
Records and weapon added.

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 4]
```diff
@@ -334,3 +334,103 @@
...
+					if(!H.sanity_lost)
+						continue
+				else
+					continue
+			L.apply_damage(special_damage, BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+			new /obj/effect/temp_visual/dir_setting/bloodsplatter(get_turf(L), pick(GLOB.alldirs))
+
+/obj/item/ego_weapon/censored/equipped(mob/user, slot, initial = FALSE)
+	. = ..()
+	RegisterSignal(user, COMSIG_MOB_APPLY_DAMGE, .proc/OnMobDamage)
```
Comment (dsmith328) [inline code/game/objects/items/ego_weapons/aleph.dm:412]: Are these signals a Spelling issue or are they actually "COMSIG_MOB_APPLY_DAMGE"?

Comment (EgorDinamit) [reply code/game/objects/items/ego_weapons/aleph.dm:412]: Were they to be a mistake, it wouldn't compile. Yes, for some reason it's really written as "DAMGE".

**Review by dsmith328 — APPROVED** [review]

**Comment by EgorDinamit** [comment]
image
@dsmith328 You forgot to ask to squash commits :)

**Comment by dsmith328** [comment]
You should squash commits so I can probably just straight up Merge this

**Comment by EgorDinamit** [comment]
PR is not ready, still lacking inhand sprites for the weapon.

**Comment by EgorDinamit** [comment]
PR is mostly ready, just lacking inhands and that's about it. This can be done later.

**Pull request merged**